src/data/constants/learnerCredit.js

```diff
@@ -0,0 +1,28 @@
+export const PAGE_TITLE = 'edX Enterprise Catalogs';
```

Comment:
I would recommend keeping the constants for LCM close to the `learner-credit-management` module/folder. We have a `/src/components/learner-credit-management/data/constants.js` file things like this could go into.